packages/slice-machine/src/legacy/lib/models/common/widgets/Group/ListItem/index.jsx

```diff
@@ -153,6 +153,13 @@ export const CustomListItem = ({
       apiId: groupItem.key,
       newKey: groupItem.key,
       value: Groups.toSM(newGroupValue),
+      groupId: groupItem.key,
+      updateMeta: {
+        fieldDeletedOrIdChanged: {
+          previousPath: [groupItem.key, key],
```

Comment:
❓ Actually where I'm lost is the group management, you're managing the group field selection, but it's not yes supported by the back-end no?

Reply:
Yeah, it's not supported, I just passed to be correct, and because I did the same for the rename action.